Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Properties/AssemblyInfo.cs

```diff
@@ -34,18 +17,6 @@
 [assembly: Dependency("System.Core", LoadHint.Always)]
 [assembly: Dependency("System.Xml", LoadHint.Always)]
 
-// WinForms
-[assembly: Dependency("System.Windows.Forms", LoadHint.Sometimes)]
-
-[assembly: Dependency("System.Drawing", LoadHint.Sometimes)]
-[assembly: Dependency("System.Configuration", LoadHint.Sometimes)]
-
-[assembly: Dependency("Windows.Web.winmd", LoadHint.Sometimes)]
```

Comment:
We still want these. Is there a case where the WINMD is not loaded when using this lib?